draft-ietf-scitt-architecture.md

```diff
@@ -480,16 +479,16 @@ The Transparency Service provides a history of Statements, which may be made by
 
 Transparency is implemented by providing a consistent, append-only, cryptographically verifiable, publicly available record of entries.
 A SCITT instance is referred to as a Transparency Service.
-Implementations of Transparency Services may protect their Append-only Log using a combination of trusted hardware, replication and consensus protocols, and cryptographic evidence.
-A Receipt is an offline, universally-verifiable proof that an entry is registered in the Append-only Log.
+Implementations of Transparency Services may protect their Verifiable Data Structure using a combination of trusted hardware, replication and consensus protocols, and cryptographic evidence.
```

Comment:
```suggestion
Implementations of Transparency Services may protect their Append-only Log and Verifiable Data Structure using a combination of trusted hardware, consensus protocols, and cryptographic evidence.
```

Replication is a kind of consensus, TS implementation that provide for example the ability to fetch Transparent Statements and not just receipts will want to protect the whole log, not just a VDS. Even implementations that don't may simply derive the VDS from a protected log.

Reply:
I don't like this change, it confuses the security mechanism for receipts, with securing mechanisms for storing VDS.

Reply:
@OR13, can you clarify if you support the current PR, or the suggested change?